src/deprecated.jl

```diff
@@ -22,3 +22,6 @@ import Base: keys, values, insert!
 
 @deprecate stackdf stackdt
 @deprecate meltdf meltdt
+
+@deprecate readtable CSV.read
```

Comment:
Come to think of it, since DataTables is new, why do we have to deprecate anything? Don't we have a clean slate, more or less?